pkg/conduit/config.go

```diff
@@ -39,47 +39,46 @@ const (
 	SchemaRegistryTypeBuiltin   = "builtin"
 )
 
-type ConfigDB struct {
-	// When Driver is specified it takes precedence over other DB related
-	// fields.
-	Driver database.DB
-
-	Type   string `long:"db.type" usage:"database type; accepts badger,postgres,inmemory,sqlite"`
-	Badger struct {
-		Path string `long:"db.badger.path" usage:"path to badger DB"`
-	}
-	Postgres struct {
-		ConnectionString string `long:"db.postgres.connection-string" usage:"postgres connection string, may be a database URL or in PostgreSQL keyword/value format"`
-		Table            string `long:"db.postgres.table" usage:"postgres table in which to store data (will be created if it does not exist)"`
-	}
-	SQLite struct {
-		Path  string `long:"db.sqlite.path" usage:"path to sqlite3 DB"`
-		Table string `long:"db.sqlite.table" usage:"sqlite3 table in which to store data (will be created if it does not exist)"`
-	}
-}
+// Config holds all configurable values for Conduit.
+type Config struct {
+	ConduitCfgPath string `long:"config.path" usage:"global conduit configuration file" default:"./conduit.yaml"`
 
-type ConfigAPI struct {
```

Comment:
Is there a special reason for reverting these changes? I don't mind either, I'm just curious if this helps something.

Reply:
@hariso it doesn't. I just happened to scroll up and down a lot and realized that it didn't really make a difference. This way is easier to see the structure in my opinion.